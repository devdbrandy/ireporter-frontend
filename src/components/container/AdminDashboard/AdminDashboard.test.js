import React from 'react';
import { shallow } from 'enzyme';
import { AdminDashboard } from './index';

/**
 * Wrapper for enzyme shallow component
 *
 * @returns {object} The mocked props and component
 */
const setup = () => {
  const props = {
    isAdmin: false,
    fetchRecords: jest.fn(),
    records: [],
    deleteRecord: jest.fn(),
    updateStatus: jest.fn(),
    overview: {},
  };

  const component = shallow(<AdminDashboard {...props} />);
  return { props, component };
};

describe('<AdminDashboard />', () => {
  it('renders without crashing given the required props', () => {
    const { component } = setup();
    expect(component).toMatchSnapshot();
  });
});
