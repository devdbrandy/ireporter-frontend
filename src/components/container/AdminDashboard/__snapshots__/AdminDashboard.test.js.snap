// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AdminDashboard /> renders without crashing given the required props 1`] = `
<Redirect
  to="/dashboard"
/>
`;
